sdk/core/Azure.Core/src/Request.cs

```diff
@@ -10,7 +10,7 @@ namespace Azure.Core.Http
 {
     public abstract class Request : IDisposable
     {
-        public virtual RequestUriBuilder UriBuilder { get; set; } = new RequestUriBuilder();
+        public virtual RequestUriBuilder Uri { get; set; } = new RequestUriBuilder();
```

Comment:
In relation to my other comment, what about just "Builder" or leave it as "UriBuilder"? I realize @KrzysztofCwalina wasn't crazy about it, but `Uri.ToUri()` looks even more strange.

Reply:
I think calling `ToUri()` is the less common case and @KrzysztofCwalina brings up how he dislikes current names all the time :)